fremantle-industries/tai
# WIP: Rename Tai.NewOrders -> Tai.Orders

- Leave facade with deprecation warnings

TODO:

- Rename IEx commands
- Rename modules
- Rename files
- Remove facade
- Rename NewOrders to Orders
- Delete old orders and replace with new orders
- Make sure create order tests make sense now
- Extract shared DateTimeFactory
- Test cleared order callbacks
- Test Services.ExecuteOrderCallback
- Last received timestamps from responses. Are they needed? Or should
that be done in the worker? Are they correctly set for orders updated
at, last received at etc...
- Add catch all order callback receiver for orders that get updated but
don't have a callback e.g. on restart after a full crash
- Ensure polymorphic_embed works with SQLite without branch
- Check that order transitions match up with state diagram
- Add postgres to test matrix

DONE:
